Assets/MapzenMap.cs

```diff
@@ -22,73 +28,60 @@ public class MapzenMap : MonoBehaviour
     private List<TileTask> pendingTasks = new List<TileTask>();
     private AsyncWorker worker = new AsyncWorker(nWorkers);
 
+    private UnityIO tileIO = new UnityIO();
+
     void Start()
     {
-        TileBounds bounds = new TileBounds(Area);
-
-        foreach (var tileAddress in bounds.TileAddressRange)
+        // Construct the HTTP request
+        for (int x = 0; x < TileRangeX; ++x)
         {
-            var wrappedTileAddress = tileAddress.Wrapped();
```

Comment:
This has been removed probably from the rebase, we don't want to use the tile range anymore so this can go.